client/src/views/uploadPage.vue

```diff
@@ -78,27 +81,72 @@ async function onCropperReady() {
 		console.log(error)
 	}
 }
+
+//checks bytes of file header to check file type because human readable MIME type can be manipulated
+function checkFileType(file) {
+  const reader = new FileReader();
+  let header = "";
+
+  reader.onload = (e) => {
+    let fileType = "";
+    let arr = new Uint8Array(e.target.result).subarray(0, 4);
```

Comment:
change this to `.subarray(0, 16` to check first 16 bytes instead, in this way, you can handle more file types